hr_employee_id/models/hr_employee.py

```diff
@@ -52,6 +52,6 @@ def _generate_identification_id(self):
 
     @api.model
     def create(self, vals):
-        eid = self._generate_identification_id()
-        vals['identification_id'] = eid
+        if not vals.get('identification_id'):
+            vals['identification_id'] = self._generate_identification_id()
```

Comment:
can't we use a `vals.setdefault('identification_id', self._generate_identification_id())` here?